infra/license/license-checker.ts

```diff
@@ -175,10 +298,17 @@ if (issueFound === false) {
   resultComment += (':heavy_check_mark: No license issue found' + EOL);
 }
 
-var resultZip = {
-  'resultComment': resultComment,
-  'warnCount': warningList.WarningCount.toString(),
-  'failCount': deniedList.DeniedCount.toString()
+let resultSet = new VerificationResultSet();
+for (const pkgName in usedLicenseList) {
+  const pkgLicense = usedLicenseList[pkgName].licenses;
+  resultSet.add(
+      {pkgName: pkgName, pkgLicense: pkgLicense, verificationResult: verify(pkgName, pkgLicense)});
+};
+
+let resultZip = {
```

Comment:
"outer code" means "outside of this code"? or "another scope in this file"?

Reply:
I mean `another scope in this file`

Reply:
It's good to minimize exported variables.

Reply:
I didn't catch your point T.T
As this will not be in namespace, function, class, I don't know how to minimize export...!
Sorry for lack of knowledge..

Reply:
Ignore my comment. I'll share my idea next time.